src/neptune_scale/__init__.py

```diff
@@ -277,63 +277,89 @@ def _create_run(
             create=CreateRun(
                 family=self._family,
                 fork_point=fork_point,
-                experiment_id=as_experiment,
+                experiment_id=experiment_name,
                 creation_time=None if creation_time is None else datetime_to_proto(creation_time),
             ),
         )
         self._operations_queue.enqueue(operation=operation)
 
+    def log_metrics(
+        self,
+        step: Optional[Union[float, int]] = None,
+        timestamp: Optional[datetime] = None,
+        data: Optional[Dict[str, float]] = None,
+    ) -> None:
+        self.log(step=step, timestamp=timestamp, metrics=data)
+
+    def log_fields(
+        self,
+        timestamp: Optional[datetime] = None,
+        data: Optional[Dict[str, Union[float, bool, int, str, datetime, list, set]]] = None,
+    ) -> None:
+        self.log(timestamp=timestamp, fields=data)
+
+    def log_string_sets(
```

Comment:
Why the shift from `tags` to `string sets`? Is there a crucial difference between tag and non-tag string sets?

I think "tags" is such a familiar concept that we shouldn't hide it under more technical terms unless there's a very good reason for it.

Reply:
My suggestion:

- `log_configs()`
- `log_metrics()`
- `add_tags()`
- `remove_tags()`

If we must use a unified method for string sets, I think `manage_tags()` is better. It mirrors what you can see in the UI and is descriptive. But a downside remains: I think it's not convention to use verbs or actions as parameter names and (`log_x(add=...)` looks clunky -> I'd propose a split if possible, as in the list above.